src/DataEntry.tsx

```diff
@@ -56,6 +59,10 @@ export default function DataEntry() {
       Playoffs: preMatch.Playoffs,
     });
 
+    if(!error) {
+      await updateAverage(settings, preMatch, auto, teleop);
+    }
+
```

Comment:
Maybe this could be moved down to be the else case of the error message if statement below?

Reply:
yeah ill move that